### DISCUSSION TITLE
Jobs triggered with workflow dispatch and workflow run are missing branch names

### DISCUSSION THREAD

**Discussion by joshowen** [community/community#14497] [Actions]
See https://github.community/t/tag-branchname-in-workflow-dispatch-workflow/134588

We would like to trigger downstream jobs like deploys using `workflow_run`, but the branch context isn't displayed in the UI.

**Comment by paresy** [comment]
Using this (https://github.blog/changelog/2022-09-26-github-actions-dynamic-names-for-workflow-runs) may be a workaround, but i also would like to have this feature. I am not yet sure how to set the run-name only for manual triggers.
